lib/sli/processes.sli

```diff
@@ -1219,26 +1219,6 @@ ifelse
 statusdict/exitcodes :: /restart_serial 4 put
 
 
-% experimental, MD 090413
-/mpirun_call 
-{
- /mpirun lookup   % better mpirun_name
- {pop}
- {
-  M_FATAL (mpirun_call) (Cannot find definition of SLI command /mpirun.) message
-  M_FATAL (mpirun_call) (Please consult ~/.nestrc to learn how to provide) message
-  M_FATAL (mpirun_call) (NEST with this command.) message
- } ifelse
-
- mpirun 0 shpawn % arguments of mpirun are identical to the ones of mpirun_call
- pop             % don't know what to do with the boolean returnd by shpawn
-
- % return value is return value of command
-}
-def
-
```

Comment:
My reasoning for the removal was like this: the check for `/mpirun` is not needed anymore, now that we always provide a not-commented version of `/mpirun` in `.nestrc`. The remaining code does not add anything above what `/mpirun` and `/nest_indirect` already provide.